packages/git/git.pacscript

```diff
@@ -1,11 +1,13 @@
 name="git"
-version="2.34.1"
+version="2.35.1"
 url="https://mirrors.edge.kernel.org/pub/software/scm/${name}/${name}-${version}.tar.gz"
 build_depends="dh-autoreconf"
 depends="libcurl4-gnutls-dev libexpat1-dev gettext libz-dev libssl-dev asciidoc xmlto docbook2x install-info"
-replace="git"
```

Comment:
Keep the `replace` variable.

Reply:
see: https://github.com/pacstall/pacstall-programs/runs/5859071549?check_suite_focus=true#step:9:26 Pacstall fails to install because of it